dsc/gettingStarted.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started with PowerShell Desired State Configuration #
+
+This guide describes how to begin creating PowerShell Desired State Configuration documents and apply them to machines. It assumes basic familiarity with PowerShell cmdlets, modules, and functions. 
+
+
+## Create a Configuration ##
+
+[**Configurations**](https://msdn.microsoft.com/en-us/powershell/dsc/configurations) are documents that describe an environment. Environments consist of "**nodes**", which are commonly virtual or physical machines. 
+
+Configurations can come in a variety of forms. The easiest way to create a new configuration is to create a .ps1 (PowerShell script) file. To do this, open your editor of choice. The PowerShell ISE is a good choice, since it understands DSC natively. Save the following as a PS1:
+
+```powershell
+configuration myFirstConfiguration
+{
+    import-dscresource -name WindowsFeature
```

Comment:
No reason not to use proper casing here:

`Import-DscResource -Name WindowsFeature`